extensions/cms/src/cmsUtils.ts

```diff
@@ -37,7 +37,7 @@ export class CmsUtils {
 	}
 
 	private _credentialProvider: azdata.CredentialProvider;
-	private _cmsService: mssql.CmsService;
+	private _cmsService: mssql.ICmsService;
```

Comment:
So we do what to use the `I` prefix for interfaces?  I see this being done inconsistently, both in our code and VS Code.  I personally prefer not to add it, since it seems like an old COM\Hungarian-notation hold-over, but I'm fine either way.  Though I would like to start being consistent at least within our code base.

Reply:
I think it depends on how its being used. If its an actual (classic) interface; like there is a class that implements it, then I think the `I` makes sense. however, in places where we have interfaces that are basically just defining the shape of an JSON, I don't think we should use `I`

Reply:
And I think this is the logic vscode uses (at least from what I can tell)

Reply:
Actually it looks like they don't have a guideline. either way I think ti makes sense to have `I` for interfaces that are meant to have hard implementation like our services, but for just defining the shape of an object (like all the interfaces in our api definitions) I don't think the `I` is providing anything.